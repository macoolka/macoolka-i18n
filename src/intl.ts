import en from './i18n/en.json';
import zh from './i18n/zh.json';
import { I18NLanguagePackage } from './types';
/**
 * The provide i18n message.
 */
const i18n: I18NLanguagePackage = {
    en,
    zh,
};
export default i18n;
