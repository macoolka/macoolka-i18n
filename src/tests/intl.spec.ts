import intl from '../intl'
describe('inner', () => {
    it('intl data', () => {
        expect(intl).toMatchSnapshot()

    })

})