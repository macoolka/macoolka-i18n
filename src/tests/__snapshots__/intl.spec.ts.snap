// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inner intl data 1`] = `
Object {
  "en": Object {
    "macoolka.i18n.errors.LanguageNotFound": "Must provide a default locale({locale})",
    "macoolka.i18n.errors.MessageNotFound": "Message ({id}) not found in locale({locale})",
  },
  "zh": Object {
    "macoolka.i18n.errors.LanguageNotFound": "缺省语言({locale})没有发现",
    "macoolka.i18n.errors.MessageNotFound": "消息({id})没有定义，请核对语言包({locale})",
  },
}
`;
